### DISCUSSION TITLE
Training Detection model on custom data

### DISCUSSION THREAD

**Discussion by gibotsgithub** [PaddlePaddle/PaddleOCR#6102] [Q&A] [answered]
I want to finetune pretrained model on my data but just after downloading the pretrained model when checking the detection prediction for any image it always show empty So not sure weather I am using pretrained weights or training the whole model from scratch can you please suggest something I just want to finetune pretrained model on my data ?

**Comment by LDOUBLEV** [comment, marked as answer]
Hi, which model do you wish to finetune？
There are three models for you to choose from, namely PPOCR, PPOCRv2, PPOCRV3. [model_list](https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.5/doc/doc_en/models_list_en.md#1-chinese-detection-model)

Take the training of the PPOCRv3 model as an example.
The PP-OCRv3 model trained on chinese data are follows.
<img width="983" alt="image" src="https://user-images.githubusercontent.com/26592129/167615669-d5936a89-e9c3-4f64-9e80-6a5df5667051.png">

The model for english.
https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.5/doc/doc_en/models_list_en.md#12-english-detection-model


And corresponding config files.
<img width="325" alt="image" src="https://user-images.githubusercontent.com/26592129/167615731-f8b55162-7abb-4920-af1b-4b1d5febdca6.png">


First, verify that the prediction can get the result after loading the pretrained weight.
```
wget https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_det_distill_train.tar
tar xf ch_PP-OCRv3_det_distill_train.tar 
# inference 
python3 tools/infer_det.py -c configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_cml.yml -o Global.pretrained_model=./ch_PP-OCRv3_det_distill_train/best_accuracy  Global.infer_img=./doc/imgs/11.jpg
```
The detected Image saved in ./checkpoints/det_db/det_results_Student/11.jpg
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/26592129/167617281-0b52e8f3-5b7e-46dd-a5eb-21cdf162a478.png">

So, `-o Global.pretrained_model=./ch_PP-OCRv3_det_distill_train/best_accuracy` is OK to load pretrained model. 


For finetune training of distillation models, please refer to [doc](https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.5/doc/doc_en/knowledge_distillation_en.md#225).
